Assets/MRTK/SDK/Experimental/Editor/Inspectors/BoundsControl/BoundsControlInspector.cs

```diff
@@ -113,16 +123,33 @@ public override void OnInspectorGUI()
                     EditorGUILayout.LabelField(new GUIContent("Visuals", "Bounds Control Visual Configurations"), EditorStyles.boldLabel, GUILayout.ExpandWidth(true));
                     using (new EditorGUI.IndentLevelScope())
                     {
+                        enabledHandles.intValue = (int)(HandleFlags)EditorGUILayout.EnumFlagsField("Enabled Handles ", (HandleFlags)enabledHandles.intValue);
 
                         showBoxConfiguration = InspectorUIUtility.DrawScriptableFoldout<BoxDisplayConfiguration>(boxDisplayConfiguration, 
                                                                                                                  "Box Configuration", 
                                                                                                                  showBoxConfiguration);
-                        showScaleHandlesConfiguration = InspectorUIUtility.DrawScriptableFoldout<ScaleHandlesConfiguration>(scaleHandlesConfiguration, 
-                                                                                                                            "Scale Handles Configuration", 
+
+                        if (((HandleFlags)enabledHandles.intValue).HasFlag(HandleFlags.Scale))
```

Comment:
Super nit: To reduce some of the noise here we could have a local helper variable i.e.

HandleFlags enabledHandleFlags = (HandleFlags)enabledHandles.intValue;

Just to cut out the visual noise associated with getting HandleFlags

Reply:
Done, thanks.